src/main/java/org/ibp/api/java/study/StudyService.java

```diff
@@ -49,22 +48,32 @@
 
 	String getProgramUUID(Integer studyIdentifier);
 
-	StudyDetailsDto getStudyDetailsForGeolocation (final Integer geolocationId);
+	StudyDetailsDto getStudyDetailsForGeolocation(final Integer geolocationId);
 
 	Long countStudies(final Map<StudyFilters, String> filters);
 
 	List<PhenotypeSearchDTO> searchPhenotypes(final Integer pageSize, final Integer pageNumber, final PhenotypeSearchRequestDTO requestDTO);
 
 	long countPhenotypes(final PhenotypeSearchRequestDTO requestDTO);
 
-	List<org.generationcp.middleware.domain.dms.StudySummary> getStudies(final Map<StudyFilters, String> filters, Integer pageSize, Integer pageNumber);
+	List<org.generationcp.middleware.domain.dms.StudySummary> getStudies(final Map<StudyFilters, String> filters, Integer pageSize,
+		Integer pageNumber);
 
-	Boolean isSampled (final Integer studyId);
+	Boolean isSampled(final Integer studyId);
 
 	List<StudyTypeDto> getStudyTypes();
-	
+
 	StudyReference getStudyReference(final Integer studyId);
 
-	void updateStudy (final Study study);
+	void updateStudy(final Study study);
```

Comment:
`StudyService `does not seem to be the best place to put these new service as these have no study identifier involved at all. Maybe use `VariableService`?